easybuild/tools/systemtools.py

```diff
@@ -108,7 +108,7 @@
 POWER = 'POWER'
 X86_64 = 'x86_64'
 RISCV32 = 'RISC-V-32'
-RISCV64 = 'RISC-V-64'
+RISCV64 = 'RISCV64'
```

Comment:
I don't mind changes this, but we should make the same change for the `RISCV32` constant, to keep it consistent?